test/cases/api.sh

```diff
@@ -293,16 +316,32 @@ case $(set +x; . /etc/os-release; echo "$ID-$VERSION_ID") in
     ;;
 esac
 
+# Only RHEL need subscription block.
+SUBSCRIPTION_BLOCK=",
+    \"subscription\": {
+      \"organization\": ${API_TEST_SUBSCRIPTION_ORG_ID:-},
+      \"activation-key\": \"${API_TEST_SUBSCRIPTION_ACTIVATION_KEY:-}\",
+      \"base-url\": \"https://cdn.redhat.com/\",
+      \"server-url\": \"subscription.rhsm.redhat.com\",
+      \"insights\": true
+    }"
+
 function createReqFileAWS() {
   AWS_SNAPSHOT_NAME=$(uuidgen)
 
+  if [[ "$ID" == "rhel" ]]; then
+    subscription="$SUBSCRIPTION_BLOCK"
+  else
+    subscription=''
+  fi
```

Comment:
You can move this condition outside the function and conditionally define the value of `SUBSCRIPTION_BLOCK`, then avoid the repetition inside every function.

Reply:
Yes you're right. Update.